docs/basics/basic_scenario.md

```diff
@@ -0,0 +1,49 @@
+# Scenario
+
+Here is the basic scenario structure.  Scenario can be one or many.  When many scenarios are specified, they will be executed sequentially.  For more detail, please see [scenario]({{ "/docs/concepts.html#scenario" | absolute_url }})
+
+Note: httpRequest is a **step**, it will be the same level as all the other STEPS.
+
+```yaml
+scenario:
+- SCENARIO_NAME:
+  (Optional - STEPS)
+  - httpRequest: ...
+```
+
+Example: 
+
+```yaml
+scenario:
+- queryFlight:
+  - thinkTime:
+      duration: 3s
+  - set: isOneWay <- false
+  - httpRequest: ...
+```
+
+## Sequences of Scenario
+
+```yaml
+scenario:
+- SCENARIO_NAME:
```

Comment:
Okay, when you use this syntax, the *sequences* will be automatically linked for serial execution (it is a common case). Non-serial execution is specified using `initialSequences` and `sequences` (add link).

Reply:
added link to initialSequences